.gitignore

```diff
@@ -3,4 +3,4 @@
 *.pyc
 
 build
-source/locale/pot/.doctrees
```

Comment:
Please undo this change.

Reply:
I must have deleted this line by mistake. Change undone

Reply:
Please run `git reset --hard upstream/manual-2.2.x -- .gitignore` to fix the missing newline at end of file, then add and commit the change.

Reply:
`git reset --hard upstream/manual-2.2.x -- .gitignore` didn't work so this is what i did. 

![image](https://user-images.githubusercontent.com/28389950/96376508-f397c880-1187-11eb-90c5-79dd63473c54.png)

I read that `git checkout HEAD -- <path>` will do a hard reset for a path, replacing both the index and the working tree with the version from the HEAD commit. 

This is what my tree looks like.
![image](https://user-images.githubusercontent.com/28389950/96376816-ce0bbe80-1189-11eb-91a8-c56a933be003.png)

So i added this repo as the upstream 
![image](https://user-images.githubusercontent.com/28389950/96376834-e67bd900-1189-11eb-8238-4b63bc9d7bdc.png)
 and repeated the hard reset with `git checkout HEAD -- <path>`

I hope it worked.... Did it work?